Develop/scripts/script.js

```diff
@@ -0,0 +1,198 @@
+$(document).ready(function () {
+  var cityArray = [];
+  var lon = "";
+  var lat = "";
+  var apiKey = "39edc21fee70b725a0dbcb6dbed85366";
+  var city = "";
+
+  retrieveCityArray();
+
+  // When user searches for city
+  $("#add-city").on("click", function (event) {
+    event.preventDefault();
+
+    storeCityArray();
+    retrieveCityArray();
+  });
+
+  // When user clicks city button
+  $(document).on("click", ".city-button", getCityClicked);
+
+  function storeCityArray() {
+    city = $("#city-input").val().trim();
+    localStorage.setItem("lastSearchedCity", JSON.stringify(city));
+
+    cityArray.push(city);
+    localStorage.setItem("cities", JSON.stringify(cityArray));
+  }
+
+  function retrieveCityArray() {
+    var retrievedArray = localStorage.getItem("cities");
+    city = JSON.parse(localStorage.getItem("lastSearchedCity"));
+
+    if (retrievedArray !== null) {
+      cityArray = JSON.parse(retrievedArray);
+      renderCityButtons();
+      getLonLat();
+    }
+  }
+
+  function renderCityButtons() {
+    $("#city-buttons-div").empty();
+
+    for (var i = 0; i < cityArray.length; i++) {
+      var newButton = $("<button>");
+
+      newButton.addClass("btn btn-outline-secondary city-button");
+      newButton.attr("city-name", cityArray[i]);
+      newButton.text(cityArray[i]);
+
+      $("#city-buttons-div").prepend(newButton);
+    }
+  }
+
+  function getCityClicked() {
+    city = $(this).attr("city-name");
+    localStorage.setItem("lastSearchedCity", JSON.stringify(city));
+    getLonLat();
+  }
+
+  function getLonLat() {
+    var queryURL =
+      "https://api.openweathermap.org/data/2.5/weather?q=" +
+      city +
+      "&appid=" +
+      apiKey;
+    $.ajax({
+      url: queryURL,
+      method: "GET",
+    }).then(function (response) {
+      lon = response.coord.lon;
+      lat = response.coord.lat;
+      getWeatherAPIInfo();
+    });
+  }
+
+  function getWeatherAPIInfo() {
```

Comment:
I would say the function name isn't accurate. You're not getting weather API Info. You're getting weather data.